quarkchain/cluster/cluster.py

```diff
@@ -89,7 +89,9 @@ def __init__(self, config, cluster_id=""):
 
     async def run(self):
         await self.run_master()
```

Comment:
Should we also run master in discovery mode?

Reply:
yes we should, since the p2p network is started inside `master.py`